doc/bum_storm_control/bum_storm_control_hld.md

```diff
@@ -0,0 +1,399 @@
+
+
+###### Table 2: Storm Control SAI attributes
+| Storm Control component  | SAI attributes                                         |                          
+|--------------------------|--------------------------------------------------------|
+| Meter type               | SAI_POLICER_ATTR_METER_TYPE                            |
+| Meter mode               | SAI_POLICER_ATTR_MODE                                  |
+| CIR (bps)                | SAI_POLICER_ATTR_CIR                                   |
+| Unknown-unicast policer  | SAI_PORT_ATTR_FLOOD_STORM_CONTROL_POLICER_ID           |
+| Broadcast policer        | SAI_PORT_ATTR_BROADCAST_STORM_CONTROL_POLICER_ID       |
+| Unknown-Multicast policer| SAI_PORT_ATTR_MULTICAST_STORM_CONTROL_POLICER_ID       |
+
+
+The BUM storm-control feature can be enabled on a physical port. 
+
+The **create_policer** SAI API is used to create the policer based on the policer parameters listed in Table 2.
+   - Meter Type - Can be packets or bytes. Storm-control uses bytes meter type. 
+   - Meter Mode - Can be st_tcm or tr_tcm or storm. Storm-control uses storm meter mode. 
+   - CIR (bps) - Value of CIR given in bps. 
+
+The **set_port_attribute** SAI API is used to set the policer on an interface. 
+   - Unknown-unicast policer - Enables storm-control for unknown-unicast traffic received on the interface. 
+   - Broadcast policer - Enables storm-control for broadcast traffic received on the interface. 
+   - Unknown-multicast policer - Enables storm-control for unknown-multicast traffic received on the interface. 
+
+## 3.5 CLI
+### 3.5.1 Data Models
+Configuration is supported using Click and Klish CLI commands.
+
+
+### 3.5.2 Configuration Commands
+This section lists down the Click commands.
+BUM storm-control can be configured only on physical interfaces.  
+**switch# config interface storm-control {broadcast | unknown-unicast | unknown-multicast} {add|del} \<interface_name\> {\<kilo_bits_per_second\>}**
+
+
+#### 3.5.2.1 Enable Broadcast storm control on a physical interface
+This command configures broadcast storm-control on a physical interface. <br>
+**switch# config interface storm-control broadcast add \<interface_name\> {\<kilo_bits_per_second\>}**
+
+
+#### 3.5.2.1 Enable Unknown-unicast storm control on a physical interface
+This command configures unknown-unicast storm-control on a physical interface. <br>
+**switch# config interface storm-control unknown-unicast add \<interface_name\> {\<kilo_bits_per_second\>}**
+
+
+#### 3.5.2.1 Enable Multicast storm control on a physical interface
+This command configures unknown-multicast storm-control on physical interface. <br>
+**switch# config interface storm-control unknown-multicast add \<interface_name\> {\<kilo_bits_per_second\>}**
+
+
+### 3.5.3 Show Commands
+The following show command displays storm-control configurations. 
+
+**show storm-control {all | interface \<interface_name\>}**
```

Comment:
Suggest to use the following format,
show storm-control [interface <interface_name>]

Reply:
Will do.